cocotb/regression.py

```diff
@@ -654,6 +654,10 @@ class TestFactory:
     the test description) includes the name and description of each generator.
     """
 
+    # Prevent PytestCollectionWarning from pytest
```

Comment:
```suggestion
    # Prevent PytestCollectionWarning from cocotb-test
```

Reply:
Strictly speaking, ``__test__`` also works with ``unittest`` and ``nose``.

Reply:
Then maybe something more general?
```suggestion
    # Prevent warnings from collection of TestFactories by unit testing frameworks.
```

Reply:
Yeah, that's a much better description.